config/config.exs

```diff
@@ -53,3 +53,7 @@ config :phoenix, :json_library, Jason
 # Import environment specific config. This must remain at the bottom
 # of this file so it overrides the configuration defined above.
 import_config "#{config_env()}.exs"
+
+# Google auth variables
+config :elixir_auth_google,
+  google_scope: "profile email https://www.googleapis.com/auth/calendar.events"
```

Comment:
Sadly, when running the `Cal` App on `localhost` using just the `"https://www.googleapis.com/auth/calendar.events"` scope I'm seeing the following error:

<img width="1788" alt="image" src="https://user-images.githubusercontent.com/194400/236052342-28af1d70-8fee-4233-9eec-750be67b1312.png">

So I've bumped it to `https://www.googleapis.com/auth/calendar` in my App: 

<img width="754" alt="image" src="https://user-images.githubusercontent.com/194400/236052530-cdba3d79-1e4f-4b90-b446-e113235fe98c.png">